server/models/connection.js

```diff
@@ -0,0 +1,21 @@
+const Sequelize = require('sequelize');
+// require('env2')('./config.env');
+// const { DB_CONFIG } = require('../../../config.js');
+
+// const { dbname, username, password, host } = DB_CONFIG;
+module.exports = new Sequelize("unicornz", "unicornz", "12345678", {
```

Comment:
import database from env.config not like that.

Reply:
I already commented those lines, this was only for testing

Reply:
I mean line 6 not true! doesn't sense to make your username and password public